src/app/compiler.js

```diff
@@ -144,7 +144,7 @@ function Compiler (editor, queryParams, handleGithubCall, updateFiles) {
   function loadInternal (url) {
     delete window.Module;
     // Set a safe fallback until the new one is loaded
-    setCompileJSON(function (source, optimize) { compilationFinished('{}'); });
+    setCompileJSON(function (source, optimize) { compilationFinished({}); });
```

Comment:
Good catch. Shouldn't it be `{ error: 'Compiler not found.' }` or something similar?